protostar/commands/test/fuzzing/strategies/strategies_test.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from protostar.commands.test.fuzzing.exceptions import SearchStrategyBuildError
+from .one_of import OneOfStrategyDescriptor
+from .felts import FeltsStrategyDescriptor
+
+
+def test_one_of_constructor_invalid_args():
+    with pytest.raises(SearchStrategyBuildError) as ex:
+        OneOfStrategyDescriptor("Not a StrategyDescriptor")  # type: ignore
```

Comment:
```suggestion
        OneOfStrategyDescriptor(NotImplemented)
```

<img width="873" alt="obraz" src="https://user-images.githubusercontent.com/3450050/188660115-0a84818b-28c9-43e0-8968-ace489e2cce5.png">

but typechecks